horace_core/sqw/@DnDBase/DnDBase.m

```diff
@@ -1,24 +1,217 @@
-classdef DnDBase < SQWDnDBase
-    
-    %DNDBASE Summary of this class goes here
-    %   Detailed explanation goes here
-    
-    properties
-        Property1
+classdef (Abstract)  DnDBase < SQWDnDBase
+    % DnDBase Abstract base class for n-dimensional DnD object
+
+    properties(Access = protected)
+        data % dnd_sqw_data instance
+    end
+
+    properties(Constant, Abstract, Access = protected)
+       NUM_DIMS
+    end
+
+    properties(Dependent)
+        filename
+        filepath
+        title
+        alatt
+        angdeg
+        uoffset
+        u_to_rlu
+        ulen
+        ulabel
+        iax
+        iint
+        pax
+        p
+        dax
+        s
+        e
+        npix
```

Comment:
Adding comments above these properties will fill out the "Property Summary" section on the Matlab help. E.g. [`PixelData` ](https://github.com/pace-neutrons/Horace/blob/master/horace_core/sqw/PixelData/%40PixelData/PixelData.m#L80-L102):
![image](https://user-images.githubusercontent.com/46960113/100614348-ea188900-330d-11eb-9297-13bb897fc0ec.png)